website/source/guides/operations/deployment-guide.html.md

```diff
@@ -264,6 +264,17 @@ The `telemetry` stanza specifies various configurations for Vault to publish met
 
 If you decide to configure Vault to publish telemtery data, you should review the [telemetry configuration section](/docs/configuration/telemetry.html) of our documentation.
 
+### High Availability Parameters
+
+New in Vault 1.0 there are additional configuration parameters for HA. Add the below configuration to the Vault configuration file:
+
+```hcl
+api_addr = {{full URL to Vault API endpoint}}
```

Comment:
This is not a new parameter and has been around since at least 0.8.

Reply:
Thanks - Chris.
It has only now started to warn - hence the update, but I will amend

Reply:
Amended

Reply:
Looks like there’s a stray bracket in this code example. Should the value be quoted?